examples/undocumented/python/library_fisher2x3.py

```diff
@@ -1,19 +0,0 @@
-#!/usr/bin/env python
-from numpy import *
-from shogun import *
-
-x=array([[20.0,15,15],[10,20,20]])
-y=array([[21.0,21,18],[19,19,22]])
-z=array([[15.0,27,18],[32,5,23]])
-
-
-parameter_list = [[x,concatenate((x,y,z),1)]]
-
-def library_fisher2x3 (table, tables):
-	pval=Statistics_fishers_exact_test_for_2x3_table(table)
```

Comment:
I removed Statistics and Math classes from swig since there is name clashing in R with these classes.
Now this example failed which only calls a function from the statistics class. I decided to remove the example. If this isn't reasonable I will bring it back + the Statistics class in swig.

Reply:
I think it is reasonable to remove this!